<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaGenerarReporte</class>
 <widget class="QDialog" name="VentanaGenerarReporte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generar reporte</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos/qtRecursosIconos.qrc">
    <normaloff>:/iconos/iconos/icono-app-dgtic.png</normaloff>:/iconos/iconos/icono-app-dgtic.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaGenerarReporte {
	background-color: #fff;
}

#labelTituloReporte {
	font-weight: bold;
	font-size: 20px;
}

#frameGeneralReporte .QPushButton{
	color: #ffffff;
	background-color: #417fd2;
	border: 1px solid rgb(0, 0, 0);
	border-radius: 6px;
	padding-bottom: 2px;
	font: 8.5pt &quot;Segoe UI Black&quot;;
}

#frameGeneralReporte .QPushButton::hover{
	background-color: #5c8bc9;
}

#frameGeneralReporte .QPushButton::pressed{
	background-color: #063c85;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameGeneralReporte">
     <property name="minimumSize">
      <size>
       <width>378</width>
       <height>333</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>378</width>
       <height>333</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QFrame" name="frameOpcionesGenerar">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QProgressBar" name="barraProgresoReporte">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>180</y>
           <width>281</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>10</y>
           <width>241</width>
           <height>151</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlOpcionesGenerarReporte">
          <item>
           <widget class="QLabel" name="labelTituloReporte">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Elige el mes y el a√±o del reporte a generar</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deFechaReporte">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QDateEdit {
    qproperty-alignment: AlignCenter;
}</string>
            </property>
            <property name="displayFormat">
             <string>MM/yyyy</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameSeccionInferiorReporte">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>95</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labelEstadoReporte">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>351</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Listo para generar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlBotonesInferioresReporte">
          <item>
           <widget class="QPushButton" name="botonGenerarReporteServicio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Generar reporte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonCancelarReporteServicio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../recursos/qtRecursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
